paimon-spark/paimon-spark-common/pom.xml

```diff
@@ -34,10 +34,22 @@ under the License.
     <name>Paimon : Spark : Common</name>
```

Comment:
We should add scala version here, because this `paimon-spark-common` will produce two versions. So we should distinguish them.

Reply:
Addressed